### PR TITLE
Arbitrum llp tracker 

### PR DESCRIPTION
**Pull request by 20xharley**

### CODE REVIEW

File: apps/llp-tracker-ui/src/config/arbitrum.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { ChainConfig } from './type';
+
+export const Config: ChainConfig = {
+  chainId: 42161,
+  chainName: 'Arbitrum',
+  etherscanName: 'ArbiScan',
+  rpcUrl: 'https://arb-mainnet.g.alchemy.com/v2/nvmp23wCDNKccCUe596f3DuNtWYKHn2p',
```
Comment (ukin-level) [inline apps/llp-tracker-ui/src/config/arbitrum.ts]: Use public rpc

File: apps/llp-tracker-ui/src/pages/LpView/LLPAndBTCPrice.tsx [hunk 2]
```diff
@@ -51,60 +48,60 @@ const syncMethod = (chartData: ChartSyncData[], active: ChartSyncActive) => {
...
-  const llpVsBtc = useQuery(queryLLPAndBTCPrice(lpAddress, start, end));
-  const { data: lvlPriceQuery } = useQuery(queryLvlPrices());
-  const lvlPrices = lvlPriceQuery?.map((t: any) => {
+}> = ({ chainId, trancheId, lpAddress, start, end }) => {
+  const chainConfig = getChainSpecifiedConfig(chainId);
+  const lvlToken = getTokenConfig(chainConfig, chainConfig.rewardToken);
+  const llpVsBtc = useQuery(queryLLPAndBTCPrice(chainId, lpAddress, start, end));
+  const { data: currentLvlPrice } = useQuery(queryLvlPrice(chainId));
+  const { data: lvlPricesResponse } = useQuery(queryLvlPrices(chainId));
+  const lvlPrices = lvlPricesResponse?.map((t: any) => {
```
Comment (ukin-level) [inline apps/llp-tracker-ui/src/pages/LpView/LLPAndBTCPrice.tsx]: Stop using 'any'

File: apps/llp-tracker-ui/src/pages/LpView/LLPAndBTCPrice.tsx [hunk 3]
```diff
@@ -51,60 +48,60 @@ const syncMethod = (chartData: ChartSyncData[], active: ChartSyncActive) => {
...
         rewardHistoryItem?.rewardPerSecond && poolInfo?.allocPoints?.length && poolInfo?.totalAllocPoint > 0n
           ? (poolInfo?.allocPoints[trancheId] * rewardHistoryItem?.rewardPerSecond * 86400n) / poolInfo?.totalAllocPoint
           : 0n;
-      let lvlPriceItem;
-      if (lvlPrices?.length) {
-        lvlPriceItem = lvlPrices?.find(
-          (l: any) =>
-            l.timestamp.getTime() >= p.timestamp * 1000 && l.timestamp.getTime() < (p.timestamp + 86400) * 1000,
-        );
+      const lvlPriceItem = lvlPrices?.find((l: any) => l.timestamp >= p.timestamp && l.timestamp < p.timestamp + 86400);
```
Comment (ukin-level) [inline apps/llp-tracker-ui/src/pages/LpView/LLPAndBTCPrice.tsx]: Stop using 'any'

Comment (20xharley) [reply apps/llp-tracker-ui/src/pages/LpView/LLPAndBTCPrice.tsx]: fixed

File: apps/llp-tracker-ui/src/pages/LpView/LLPAndBTCPrice.tsx (hunk 2 above)
Comment (20xharley) [reply apps/llp-tracker-ui/src/pages/LpView/LLPAndBTCPrice.tsx]: fixed

File: apps/llp-tracker-ui/src/config/arbitrum.ts (hunk 1 above)
Comment (20xharley) [reply apps/llp-tracker-ui/src/config/arbitrum.ts]: fixed

**Review by 0xhari-s — APPROVED** [review]

**Pull request merged**